gradle/project.go

```diff
@@ -0,0 +1,84 @@
+package gradle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/bitrise-io/depman/pathutil"
+	"github.com/bitrise-io/go-utils/log"
+	glob "github.com/ryanuber/go-glob"
+)
+
+// Project ...
+type Project struct {
+	location string
+	monoRepo bool
+}
+
+// NewProject ...
+func NewProject(location string) (Project, error) {
+	buildGradleFound, err := pathutil.IsPathExists(filepath.Join(location, "build.gradle"))
+	if err != nil {
+		return Project{}, err
+	}
+
+	if !buildGradleFound {
+		return Project{}, fmt.Errorf("no build.gradle file found in (%s)", location)
+	}
+
+	root := filepath.Join(location, "..")
+
+	files, err := ioutil.ReadDir(root)
+	if err != nil {
+		return Project{}, err
+	}
+
+	projectsCount := 0
+	for _, file := range files {
+		if file.IsDir() {
+			e, err := pathutil.IsPathExists(filepath.Join(root, file.Name(), "build.gradle"))
+			if err != nil {
+				return Project{}, err
+			}
+			if e {
+				projectsCount++
+			}
+		}
+	}
+
+	return Project{location: location, monoRepo: (projectsCount >= 2)}, nil
+}
+
+// SetModule ...
+func (proj Project) SetModule(module string) Module {
```

Comment:
SetModule -> NewModule / CreateModule

Reply:
SetModule is because I don't want to create a new module, but want to set a module that will be used